src/lib/components/event/event-shortcut-keys.svelte

```diff
@@ -20,49 +21,51 @@
 <Drawer
   {open}
   onClick={onClose}
-  title={translate('common.keyboard-shortcuts')}
   closeButtonLabel={translate('common.close-keyboard-shortcuts')}
+  class="w-[35vw] min-w-min max-w-fit"
 >
-  <div
-    class="grid grid-cols-1 gap-8 text-gray-500 dark:text-gray-400 md:grid-cols-2 xl:grid-cols-4"
-  >
-    <div class="cell">
-      {translate('common.next-page')}
-      <div class="shortcut">
-        <Shortcut
-          arrow="right"
-          title={translate('common.arrow-key-right')}
-        />|<Shortcut title="">L</Shortcut>
+  <DrawerContent title={translate('common.keyboard-shortcuts')}>
+    <div
+      class="grid grid-cols-1 gap-8 text-gray-500 dark:text-gray-400 md:grid-cols-2 xl:grid-cols-4"
+    >
```

Comment:
Maybe `<DrawerContent />` should take a class and we can remove this extra `<div />` ?

Reply:
Hmm that would kind of be confusing I think. 

Because it could target the title or the content, this just targets the content and essentially maintains the previous API. 

Could also switch title to be it's own slot and then it would get the same semantics but I was lazy and didn't want to make this super flexible/correct because it would induce change in the calling components and also we don't use it all over and didn't want to establish a different paradigm.

Happy to switch it to `<DrawerContent /> and <DrawerTitle />`